projects/cashmere/src/lib/tabs/tab-set.component.ts

```diff
@@ -30,6 +34,9 @@ export class TabSetComponent implements AfterContentInit {
 
     @ContentChildren(TabComponent) _tabs: QueryList<TabComponent>;
 
+    /** Emits when the selected tab is changed */
+    @Output() selectChange: EventEmitter<TabChangeEvent> = new EventEmitter();
```

Comment:
I am not a fan of "selectChange" as the event. What about "selectedTabChange" or "tabSelected"?
Angular Material - "selectedTabChange"
https://material.angular.io/components/tabs/api

Kendo UI - "tabSelect"
https://www.telerik.com/kendo-angular-ui/components/layout/tabstrip/

Reply:
works for me - I like `selectedTabChange` mostly because it feels more clearly differentiated from the `tabClick` event on TabComponent